automol/util/ring.py

```diff
@@ -42,7 +34,7 @@ def cycle(items: List[object], count: int = 1) -> List[object]:
     return tuple(itertools.islice(cycler, nitems))
 
 
-def edges(items: List[object]) -> List[object]:
+def edges(items: list[object]) -> list[object]:
```

Comment:
See above. Also, the return type on this one needs to be corrected since it is returning pairs:
```
(items: Sequence[object]) -> tuple[tuple[object, object], ...]:
```